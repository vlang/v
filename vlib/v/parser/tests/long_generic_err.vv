fn long_generic[Abc]() {}
