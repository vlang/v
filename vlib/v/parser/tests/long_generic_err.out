vlib/v/parser/tests/long_generic_err.vv:1:17: error: generic parameter name needs to be exactly one char
    1 | fn long_generic[Abc]() {}
      |                 ~~~
